packages/http/src/__tests__/http-prism-instance.spec.ts

```diff
@@ -158,51 +164,32 @@ describe('Http Client .request', () => {
 
         it('fails the operation', () =>
           expect(prism.request(request, resources, config)).rejects.toThrowError(
-            ProblemJsonError.fromTemplate(NO_PATH_MATCHED_ERROR),
+            ProblemJsonError.fromTemplate(NO_PATH_MATCHED_ERROR)
           ));
       });
 
-      describe('path is valid and baseUrl is not set', () => {
-        it('fallbacks to a server from the spec', async () => {
-          const oasBaseUrl = 'http://example.com/api';
-          const reply = 'some demo reply';
-          nock(oasBaseUrl)
-            .get('/pet')
-            .reply(200, reply);
-
-          const result = await prism.request(
-            {
-              method: 'get',
-              url: {
-                path: '/pet',
-              },
-            },
-            resources,
-            config,
-          );
-
-          expect(result.output).toBeDefined();
-          expect(result.output.statusCode).toEqual(200);
-          expect(result.output.body).toEqual(reply);
-        });
-      });
-
       describe('Prism user-agent header', () => {
         describe('when the defaults are used', () => {
           it('should use Prism/<<version>> for the header', async () => {
-            const userAgent = await checkUserAgent(config, prism, resources);
+            const userAgent = await checkUserAgent(config, prism, resources, {}, 'https://stoplight.io');
 
-            expect(userAgent).toBe(`Prism/${prismVersion}`);
+            expect(userAgent).toContain(`Prism/${prismVersion}`);
           });
         });
 
         describe('when user-agent is being overwritten', () => {
           it('should have user specified string as the header', async () => {
-            const userAgent = await checkUserAgent(config, prism, resources, {
-              'user-agent': 'Other_Agent/1.0.0',
-            });
+            const userAgent = await checkUserAgent(
+              config,
+              prism,
+              resources,
+              {
+                'user-agent': 'Other_Agent/1.0.0',
+              },
+              'https://stoplight.io'
+            );
 
-            expect(userAgent).toBe('Other_Agent/1.0.0');
+            expect(userAgent).toContain('Other_Agent/1.0.0');
```

Comment:
ditto

Reply:
I have no idea why I changed this. I am so old.